### PR TITLE
Add styles to tables in details-pane

### PR DESCRIPTION
**Pull request by zzmedia**
- added styles to tables in details-pane

https://www.openproject.org/work_packages/14795

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-styleguide-details-pane-table.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by manwithtwowatches** [comment]
@zzmedia The attachments and relations tables have different numbers of columns, so i'm not sure that setting the column width percentages explicitly will work. we might have to move them out and specify them for each table separately.

**Comment by manwithtwowatches** [comment]
@zzmedia ...but i will take care of this.

**Comment by zzmedia** [comment]
@manwithtwowatches Great thx!

File: public/templates/work_packages/tabs/_attachments_table.html [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-  <tbody>
+<table ng-if="attachments.length" class="detail-pane-table attachments-table">
+  <thead>
     <tr>
-      <th>{{ I18n.t('js.label_filename')}}</th>
-      <th>{{ I18n.t('js.label_filesize')}}</th>
-      <th>{{ I18n.t('js.label_uploaded_by')}}</th>
-      <th>{{ I18n.t('js.label_description')}}</th>
-      <th>{{ I18n.t('js.label_date')}}</th>
+      <td>{{ I18n.t('js.label_filename')}}</td>
```
Comment (hschink) [inline public/templates/work_packages/tabs/_attachments_table.html]: We need to set the `title` attribute or we have no chance to read larger table headings.

File: public/templates/work_packages/tabs/_work_package_relations.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <div class="content" ng-show="expand">
     <div class="workpackages">
       <div ng-if="handler.relations">
-        <table>
+        <table class="detail-pane-table relations-table">
           <thead>
             <tr>
               <td>{{ I18n.t('js.work_packages.properties.subject') }}</td>
```
Comment (hschink) [inline public/templates/work_packages/tabs/_work_package_relations.html]: I guess applying the `title` attribute should be done for this table too.

**Comment by hschink** [comment]
There is only a minor issue on two tables. As soon as this issue is addressed and the PR is green again this can be merged.

**Pull request merged**